cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -102,6 +102,19 @@ data:
 {{- end }}
 
   {{- if and (not .Values.agent) (not .Values.cloudAgent) (not (eq (include "aggregator.deployMethod" .) "disabled")) }}
+  {{- if and (.Values.kubecostDeployment) (.Values.kubecostDeployment.queryServiceReplicas) (gt (.Values.kubecostDeployment.queryServiceReplicas | toString | atoi) 0) }}
+    upstream queryservice {
+        server {{ .Release.Name }}-query-service-load-balancer.{{ .Release.Namespace }}:9003;
+    }
+  {{- end }}
```

Comment:
The diff shows that you're adding in some lines here referencing `queryServiceReplicas`. Was that intentional? My understanding was that QSR is no more?

Reply:
Nope, unintentional, that was a mistake when I was resolving merge conflicts, thanks for the catch

Reply:
Addressed